docs/docs/stories.mdx

```diff
@@ -18,23 +18,23 @@ Here's an example of a dialogue in the Rasa story format:
 stories:
```

Comment:
Above this under "format", we say that the inputs are expressed as intents and entities, and responses/actions as action names, which is not completely true anymore